spec.html

```diff
@@ -415,7 +416,7 @@ <h1>Iterator.prototype.flatMap ( _mapper_ )</h1>
               1. Let _value_ be ? IteratorValue(_next_).
               1. Let _mapped_ be Completion(Call(_mapper_, *undefined*, « _value_, 𝔽(_counter_) »)).
               1. IfAbruptCloseIterator(_mapped_, _iterated_).
-              1. Let _innerIterator_ be Completion(GetIteratorFlattenable(_mapped_)).
+              1. Let _innerIterator_ be Completion(GetIteratorFlattenable(_mapped_, ~reject-strings~)).
```

Comment:
Passing an enum here works, but it seems easier to just hoist the typechecks to the callers.

Reply:
Yeah we could do that too. For now, I'm just interested in reflecting the semantics we want. I did it this way because I recall we wanted to reuse this AO for `AsyncIterator.prototype.flatMap`, but we can always refactor it in the stage 4 PR.